packages/sdks/src/components/content/components/enable-editor.lite.tsx

```diff
@@ -457,6 +463,13 @@ export default function EnableEditor(props: BuilderEditorProps) {
         className={getWrapperClassName(
           props.content?.testVariationId || props.content?.id
         )}
+        style={{
+          display:
+            !props.builderContextSignal.value.content &&
+            (isPreviewing() || isEditing())
```

Comment:
should this be
```jsx
style={(!props.builderContextSignal.value.content && !isPreviewing() && !isEditing())
  ? { display: 'none' }
  : {}
}

```

Reply:
because we have custom event listeners on the div ref that enable visual editing, we want to visually hide this wrapper `div` in editing/preview if content is empty, but still render it. if we don't hide it it might break customer layouts because it'll be an empty `div` with no content.

so:

- content exists: render div and `display: undefined`
- content does not exist but isEditing/isPreviewing: render div and `display: 'none'`
  -  once inline editing kicks in, it will populate the content and re-render, so `display` style will be removed

your suggestion would display this div in the second case even if we never get content, which we want to avoid.

Reply:
added, please take a look.